app/cli/import-data.py

```diff
@@ -674,20 +671,25 @@ def _do_import(db, input_dir, project_id):
     all_files = sorted(glob.glob(os.path.join(input_dir, "*", "*", "*.json")))
 
     print("importing agents")
+    import_agents(curate.default_agents(), db)
     for file_path in all_files:
         with open(file_path) as f:
             data = json.load(f)
-            import_agents(data, db)
+            possible_data = [
+                d for d in data if {"Person", "Organization"} & set(d.get("@type", {}))
```

Comment:
Minor, I find it more readable with:
```suggestion
                d for d in data if d.get("@type") in {"Person", "Organization"}
```

Reply:
`@type` can be a a list so I need to check for any overlap